resources/rds/v1alpha1/db-instance.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBInstance
+metadata:
+  name: $DB_INSTANCE_ID 
+spec:
+  allocatedStorage: 5 
+  autoMinorVersionUpgrade: false
+  availabilityZone: eu-west-1a 
+  backupRetentionPeriod: 14 
+  copyTagsToSnapshot: false
+  dbInstanceClass: db.t3.micro 
+  dbInstanceIdentifier: $DB_INSTANCE_ID 
+  dbName: $DB_NAME 
+  dbSubnetGroupName: $DB_SUBNET_GROUP_NAME 
+  dbSecurityGroups:
+  - $DB_SECURITY_GROUP_NAME 
```

Comment:
This was removed from the CRD: https://github.com/aws-controllers-k8s/rds-controller/pull/80